lib/jekyll-github-metadata/repository.rb

```diff
@@ -64,6 +64,10 @@ def repo_pages_info_opts
         end
       end
 
+      def owner_display_name
+        owner_metadata.name
```

Comment:
```suggestion
        @owner_metadata.name
```

Should this be `@` for consistency with the code below it? Unless it is different for a reason?

```
def owner_metadata
        @owner_metadata ...
```

Reply:
Oh nevermind. I see it uses the attribute wrapper `owner_metadata`, so we set `@owner_metadata` if not set already.